packages/ui/src/common/api/schemas/schema.graphql

```diff
@@ -46,8 +38,8 @@ type ApplicationFormQuestionAnswerConnection {
 }
 
 input ApplicationFormQuestionAnswerCreateInput {
-  applicationId: ID!
-  questionId: ID!
+  application: ID!
```

Comment:
Is there any reason not to use `applicationId` or `questionId`? For me `...Id` represents ID, but without ID is object

Reply:
The schema is defined externally. Unfortunately, we have to live with that as we're the consumers here :)